umbraco-heartcore/getting-started/webhooks.md

```diff
@@ -22,7 +22,7 @@ Clicking **Create Webhook** will open the creation menu on the right side.
 
 From here you add the URL that the webhook should call and select the **Event** that should trigger the webhook. Lastly, you can choose a Content Type you wish the webhook to trigger on.
 
-Do note that you will have to choose the type of event first and that this field is not mandatory.
+Please be aware that selecting the event type is the initial step, and this field is not mandatory.
```

Comment:
perhaps an info note might be better in this case?
```suggestion
{% hint style="info" %}

Selecting the event type is the initial step. Also, this field is not mandatory.

{% endhint %}

```

Reply:
Seems good to me 😄